.cspell.json

```diff
@@ -21,20 +21,36 @@
     "ingressgateway",
     "istioctl",
     "istiod",
+    "konstraint",
     "kubeval",
     "millis",
     "mkilled",
     "nindent",
     "nslookup",
+    "oo",
+    "oo mkilled",
+    "pspallowedusers",
+    "pspapparmor",
+    "pspcapabilities",
+    "pspforbiddensysctls",
+    "psphostfilesystem",
+    "psphostnetworkingports",
+    "psphostsecurity",
+    "pspprivileged",
+    "pspseccomp",
+    "pspselinux",
```

Comment:
I believe these can go after renaming the policies

Reply:
@Morriz  Adjusting policy names with a "-" (dash) implies changing some parts of the build script and Konstraint library does not support dashes when generating filenames. it would be pretty messy and I don't know if that can be predictable all the way. 
- Will try anyway

Reply:
policies are supporting dashes in the names now